go.mod

```diff
@@ -343,3 +343,5 @@ replace filippo.io/edwards25519 => filippo.io/edwards25519 v1.0.0-beta.3
 // fixes deprecation warnings and keychain undefined bugs on macOS
 // See https://github.com/99designs/keyring/issues/94
 replace github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
+
+replace github.com/smartcontractkit/ocr2vrf v0.0.0-20220622011836-190aa7f5b481 => ../ocr2vrf
```

Comment:
I guess let us know what the changes were locally, but even in a branch there should be a comment here or something to track it so we don't forget and merge this into develop.

Reply:
Good point, gonna put a "do not merge" label on the `ocr2vrf-prepub` branch for now.